site/content/blog/2022-03-22-the-stackhawk-zap-fund/index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "The StackHawk ZAP Fund"
+summary: >
+  StackHawk has launched a $100,000 ZAP Fund dedicated to improving ZAP and the ZAP Community.
+images:
+- https://www.zaproxy.org/blog/2022-03-22-the-stackhawk-zap-fund/images/hawk_bot.png
+type: post
+tags:
+- blog
+date: "2022-03-22"
+authors: 
+    - simon
+---
+
+In case you missed the news, StackHawk has [launched](https://www.stackhawk.com/blog/the-zap-fund/) a $100,000 
+[ZAP Fund](https://www.stackhawk.com/zap-fund/) dedicated to improving ZAP and the ZAP Community!
+
+![StackHawk and ZAPBot](images/hawk_bot.png)
+
+Obviously the ZAP Core Team is very grateful for this initiative and have been working with [StackHawk](https://www.stackhawk.com) to make the most of this great opportunity.
+
+As you will see from the main [ZAP Fund](https://www.stackhawk.com/zap-fund/) page there are a set of ZAP issues which now have bounties on them, including some “rolling” issues which we will keep adding until the relevant features have all been implemented.
+
+## Getting Started
+
+In order to work on any of these issues you will need to set up a ZAP development environment.
```

Comment:
Ummm should set up be all one word here? "setup"

Reply:
It looks like it should be 2 words, according to https://grammarist.com/spelling/set-up-vs-setup/ :D

Reply:
Fair, I was waffling about it that's why I asked 😀